package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@google-cloud/connect-datastore",
   "description": "Google Cloud Datastore session store for Express/Connect",
-  "version": "3.0.1",
+  "version": "3.1.0",
```

Comment:
this all happens automatically, if you don't mind reverting the version bump :+1: 

https://www.conventionalcommits.org/en/v1.0.0-beta.4/

Reply:
Of course, that makes more sense. Will do.